config/locales/en.yml

```
@@ -89,11 +89,19 @@ en:
           attributes:
             expires_at:
               inclusion: "must be in the future"
+        org:
```

This needs to be updated because of the rename to organization.

```suggestion
        organization:
```

---

thank you!